JASP-Desktop/html/js/jaspwidgets.js

```diff
@@ -531,7 +537,8 @@ JASPWidgets.NoteBox = JASPWidgets.View.extend({
 		this.$quillToolbar.querySelector('button.ql-bold').setAttribute('title', 'Bold');
 		this.$quillToolbar.querySelector('button.ql-italic').setAttribute('title', 'Italic');
 		this.$quillToolbar.querySelector('button.ql-underline').setAttribute('title', 'Underline');
-		this.$quillToolbar.querySelector('button.ql-link').setAttribute('title', 'Link');
+		this.$quillToolbar.querySelector('button.ql-link').setAttribute('title', 'Add Link');
+		this.$quillToolbar.querySelector('button.ql-image').setAttribute('title', 'Select Image');
```

Comment:
How could we translate these strings?

Reply:
I'll check out how the other strings in the results are translated.

Reply:
The menu is translated because it comes from QML, but no javascript strings are translated. We have to think of a way to translate them. Maybe we can extend the globSet object with a translation array with all strings used in javascript. This translation array should be then updated each time the translation is changed.

Reply:
We could add all the strings necessary in Javascript in the "jasp" object definition in QML? 
That way we barely need to write any code, javascript just needs to use those strings as `jasp.blablaString`

Reply:
Yup, that seems straightforward